From f64164ca8647104e02cb14f49d3620519c19f6c4 Mon Sep 17 00:00:00 2001
From: Andrey Bychkov <mrdrew@altlinux.org>
Date: Wed, 13 Feb 2019 11:52:16 +0300
Subject: [PATCH] Fix no return statement in the non-void function (according
 g++8)

---
 opennx/LibUSB.cpp      | 2 ++
 opennx/ModuleManager.h | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/opennx/LibUSB.cpp b/opennx/LibUSB.cpp
index 20be493..5ffdd9f 100644
--- a/opennx/LibUSB.cpp
+++ b/opennx/LibUSB.cpp
@@ -302,6 +302,8 @@ static bool addDeviceInfo( const USBInfo& u )
         return false;
 
     dbDeviceInfo[u.key()] = u;
+
+    return true;
 }
 // ----------------------------------------------------------------------------
 static USBInfo findDeviceInfo( const wxString &vendorID, const wxString &deviceID )
diff --git a/opennx/ModuleManager.h b/opennx/ModuleManager.h
index 493d3a3..5c4f870 100644
--- a/opennx/ModuleManager.h
+++ b/opennx/ModuleManager.h
@@ -16,7 +16,7 @@ private:
     ModulesMap modules;
 
     ModuleManager(ModuleManager const&) {} //delete
-    ModuleManager& operator= (ModuleManager const&) {} //delete
+    // ModuleManager& operator= (ModuleManager const&) {} //delete
 public:
 
     static ModuleManager& instance();
-- 
2.19.2

